### PR TITLE
CSA-44:  implemented endpoint to create a new user account

### PR DESCRIPTION
**Pull request by vitayushchyk**

### CODE REVIEW

File: career_skill_atlas/urls.py [hunk 1]
```diff
@@ -15,8 +15,16 @@
...
-from django.urls import path
+from django.urls import path, include
+
+api_patterns = [
+    path('users/', include(('user.urls', 'user'))),
+]
 
 urlpatterns = [
+
+    #ToDo(Vita): clarify, do we need prefix for API urls, like: path('api/v1/', include(api_patterns)),
```
Comment (ssliadnievv) [inline career_skill_atlas/urls.py]: yes, let's add, but without v1

File: career_skill_atlas/urls.py [hunk 2]
```diff
@@ -15,8 +15,16 @@
...
+    path('', include(api_patterns)),
     path('admin/', admin.site.urls),
+    path('api-auth/', include('rest_framework.urls', namespace='rest_framework')),
```
Comment (ssliadnievv) [inline career_skill_atlas/urls.py]: why do we need it?

File: user/models.py [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 from django.db import models
+
 from user.managers import CustomUserManager
 
 
 class User(AbstractUser):
     first_name = models.CharField(max_length=30, blank=True, null=True)
     last_name = models.CharField(max_length=50, blank=True, null=True)
-    avatar = models.ImageField(upload_to='user_avatar', blank=True)
+    avatar = models.ImageField(upload_to='media/user_avatar', blank=True)
```
Comment (ssliadnievv) [inline user/models.py]: drop `media/`, better add these:
```
MEDIA_URL = 'media/'
MEDIA_ROOT = '/media'
```
to setting

File: user/migrations/0002_alter_user_avatar.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.3 on 2024-03-21 14:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (ssliadnievv) [inline user/migrations/0002_alter_user_avatar.py]: can drop

**Review by ssliadnievv — CHANGES_REQUESTED** [review]

File: career_skill_atlas/urls.py (hunk 2 above)
Comment (vitayushchyk) [reply career_skill_atlas/urls.py]: this is from drf ofical tutorial to get posibility to make requests from DRF UI, which requare auth. Probably will need it future

Comment (ssliadniev) [reply career_skill_atlas/urls.py]: @vitayushchyk Ivan already implemented endpoints for auth, we don't have other endpoints according architecture, so please drop it

**Review by ssliadniev — APPROVED** [review]

**Pull request merged**